src/components/authentication/CreateAccountPage.tsx

```diff
@@ -14,19 +14,20 @@ type CreateAccountPageProps = {
   navigation: CreateAccountPageNavigationProp;
 };
 
+const styles = StyleSheet.create({
+  root: {
+    flex: 1,
+    flexDirection: "column",
+    justifyContent: "center",
+    alignItems: "center",
+  },
+});
+
```

Comment:
Should StyleSheet be placed in a separate file? Not familiar with it to be honest, but it feels like separation of concerns is appropriate here.

Reply:
So for a majority of the styles used for StyleSheet.create like this are mainly for layout styles. If we weren't using restyle for theming, I would agree that we should be maintaining separate files for each page. But considering there is so little code generally since layout doesn't require much styling, I personally think it is okay to leave it in the tsx files